maat/FATES_Lin2015/plot_MAAT_FATES_results.py

```diff
@@ -0,0 +1,62 @@
+# This script plots the results from the MAAT toolbox
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+
+# *** READ IN DATA FROM MAAT SIMULATION [MAKE SURE FOLDER PATH IS ACCURATE] ***
+# df = pd.read_csv("/Users/emy/Desktop/Tests/MAAT/_testing_FATES/results/2023-05-30/out_FATES.csv" , usecols=['leaf.ca_conc','A'])
+df = pd.read_csv("/Users/emy/Desktop/Tests/MAAT/_testing_FATES/results/2023-05-30/out_LinInputs_FATES.csv" , usecols=['leaf.ca_conc','leaf.temp','A'], na_values='          NA')
```

Comment:
We fixed the file paths in the CLM5 version, update here and below.

Reply:
This has been corrected now (see here: https://github.com/rubisco-sfa/MLEcohydrology/commit/4f27243cae02e82cdccb0b8a9a7effe0dea65843)